linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_admin_space_left_action/ansible/shared.yml

```diff
@@ -9,7 +9,7 @@
   lineinfile:
     dest: /etc/audit/auditd.conf
     line: "admin_space_left_action = {{ var_auditd_admin_space_left_action }}"
-    regexp: "^admin_space_left_action*"
+    regexp: "^admin_space_left_action =*"
```

Comment:
This is not ideal yet, the expression in the issue looks better.

`^\s*admin_space_left_action\s*=`
`^\s*space_left_action\s*=`